styles.css

```diff
@@ -589,3 +597,122 @@ ul.list-items.skill-items {
 .button-5 {
   padding-left: 10px;
 }
+
+form {
+  display: block;
+  margin: 0;
+}
+
+.form-class {
+  display: flex;
+  flex-direction: column;
+  margin-top: 12px;
+  align-items: center;
+}
+
+.form-btn {
+  color: #6070ff;
+  background-color: #fff;
+  border-radius: 8px;
+  padding: 12px 16px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border: 1px;
+  margin-top: 12px;
+  cursor: pointer;
+}
+
+.form-btn:hover {
+  color: #fff;
+  background-color: #6070ff;
+  border: 2px solid #fff;
+}
+
+.contact-info {
+  display: flex;
+  align-items: center;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 327px;
+  height: 48px;
+  margin-top: 12px;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  padding-left: 10px;
+}
+
+.text-input {
+  font-family: "poppins", sans-serif;
+  font-size: 17px;
+  font-weight: 400px;
+  width: 327px;
+  height: 170px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin-top: 12px;
+  padding-left: 10px;
+}
+
+.Contact-paragraph {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #ebebff;
+  margin-top: 12px;
+  font-style: normal;
+  font-family: "poppins", sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+}
+
+.Contact-me p {
+  /* display: block; */
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  /* margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0;
+  margin-inline-end: 0; */
+}
```

Comment:
[OPTIONAL] Please remove the commented code from your codebase it will make your project look messy. If you need to review a previous version of your code, you can always use git history.